cairo/src/font/font_extents.rs

```diff
@@ -27,6 +27,26 @@ impl FontExtents {
     pub fn max_y_advance(&self) -> f64 {
         self.0.max_y_advance
     }
+
+    pub fn set_ascent(&mut self, ascent: f64) {
```

Comment:
Can you move the addition of the setters into a separate commit (in the same PR)?